src/containers/ViewerPanel/index.tsx

```diff
@@ -505,17 +490,36 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
     };
 
     private get embedDisplaySettings() {
-        const { belowControlsHeight, belowControlsWidth, belowScaleBarWidth } =
-            this.state.embedDisplayBreakpoints;
-        return {
-            minimalPlaybackControls: belowControlsWidth,
-            minimalCameraControls: belowControlsHeight,
-            onlyRenderBottomControls: belowControlsWidth && belowControlsHeight,
-            showScaleBar:
-                !belowScaleBarWidth &&
-                (!belowControlsWidth || !belowControlsHeight),
-            showCameraControls: !belowControlsWidth || !belowControlsHeight,
-        };
+        const { height, width } = this.state;
+        const belowControlsHeight = height <= CONTROLS_MIN_HEIGHT;
+        const belowControlsWidth = width <= CONTROLS_MIN_WIDTH;
+        const showScaleBar = width > SCALE_BAR_MIN_WIDTH;
```

Comment:
```suggestion
        const showScaleBar = width > SCALE_BAR_MIN_WIDTH && (!belowControlsWidth || !belowControlsHeight);
```

Your changes are definitely an improvement, but without this bit of logic the scale bar ends up on top of the controls sometimes.